src/Hedgehog/Hedgehog.fsproj

```diff
@@ -1,9 +1,6 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard1.6</TargetFramework>
-    <AssemblyVersion>0.8.3.0</AssemblyVersion>
-    <FileVersion>0.8.3.0</FileVersion>
```

Comment:
I guess it's safe to take those out as the version is still specified in the paket.template(?)

Reply:
Sorry. It was my mistake to delete this lines. I wanted to reset the version after generating nuget package locally and I accidentely deleted the lines. I already reverted this back.